homeassistant/components/stream/__init__.py

```diff
@@ -202,7 +215,7 @@ def _stop(self):
 
 async def async_handle_record_service(hass, call):
     """Handle save video service calls."""
-    stream_source = call.data[CONF_STREAM_SOURCE]
```

Comment:
This feels weird. Seems like this should be a camera service now that uses entity ID to target streams.

Reply:
There is already a `camera` service that called this service.  I've removed this service entirely, and just made it a method on `Stream`.